examples/HotelAndTravel/AddThingsToDoAd.php

```diff
@@ -0,0 +1,337 @@
+            'advertising_channel_type' => AdvertisingChannelType::TRAVEL,
+            'advertising_channel_sub_type' => AdvertisingChannelSubType::TRAVEL_ACTIVITIES,
+            'travel_campaign_settings'
+                => new TravelCampaignSettings(['travel_account_id' => $thingsToDoCenterAccountId]),
+            // Recommendation: Set the campaign to PAUSED when creating it to prevent
+            // the ads from immediately serving. Set to ENABLED once you've added
+            // targeting and the ads are ready to serve.
+            'status' => CampaignStatus::PAUSED,
+            // Sets the bidding strategy to MaximizeConversionValue. Only this type can be used
+            // for Things to do campaigns.
+            'maximize_conversion_value' => new MaximizeConversionValue(),
+            // Sets the budget.
+            'campaign_budget' => $budgetResourceName,
+            // Configures the campaign network options. Only Google Search is allowed for
+            // Things to do campaigns.
+            'network_settings' => new NetworkSettings(['target_google_search' => true])
+        ]);
+        // [END add_things_to_do_ad_1]
+
+        // Creates a campaign operation.
+        $campaignOperation = new CampaignOperation();
+        $campaignOperation->setCreate($campaign);
+
+        // Issues a mutate request to add campaigns.
+        $campaignServiceClient = $googleAdsClient->getCampaignServiceClient();
+        $response = $campaignServiceClient->mutateCampaigns($customerId, [$campaignOperation]);
+
+        /** @var Campaign $addedCampaign */
+        $addedCampaign = $response->getResults()[0];
+        printf(
+            "Added a Things to do campaign with resource name '%s'.%s",
+            $addedCampaign->getResourceName(),
+            PHP_EOL
+        );
+
+        return $addedCampaign->getResourceName();
+    }
+    // [END add_things_to_do_ad]
+
+    /**
+     * Creates a new ad group in the specified Things to do campaign.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $campaignResourceName the resource name of campaign that a new ad group will
+     *     belong to
+     * @return string the resource name of the newly created ad group
+     */
+    // [START add_things_to_do_ad_2]
+    private static function addAdGroup(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $campaignResourceName
+    ) {
+        // Creates an ad group.
+        $adGroup = new AdGroup([
+            'name' => 'Earth to Mars Cruise #' . Helper::getPrintableDatetime(),
+            // Sets the campaign.
+            'campaign' => $campaignResourceName,
+            // Sets the ad group type to TRAVEL_ADS. This cannot be set to other types.
+            'type' => AdGroupType::TRAVEL_ADS,
+            'cpc_bid_micros' => 10000000,
```

Comment:
Just double-checking: does setting this when the bidding strategy is MaximizeConversionValue make sense? Is it just capping the max bid?

Reply:
Thanks for catching. I think it makes sense to add it here either. The code works without this.